drivers/gpio/Kconfig

```diff
@@ -59,6 +59,22 @@ config GPIO_HOGS_INIT_PRIORITY
 	  GPIO hogs initialization priority. GPIO hogs must be initialized after the
 	  GPIO controller drivers.
 
+config GPIO_HOGS_MASK_OUTPUT
+	bool "GPIO hogs mask output"
+	default 0
```

Comment:
default n?

Reply:
The default should just be left out. It defaults to n by default 😄